modules/ROOT/pages/using-the-library.adoc

```diff
@@ -0,0 +1,40 @@
+= Using the library
+:description: Learn how to use the Neo4j GraphQL Library with your technology stack.
+
+== Library features
+
+The Neo4j GraphQL Library features:
+
+- Automatic generation of xref::queries-aggregations/queries.adoc[Queries] and xref::mutations/index.adoc[Mutations] for CRUD interactions.
+- xref::/types/index.adoc[Types], including temporal and spatial.
+- Support for both node and relationship properties.
+- Extensibility through the xref::/directives/custom-logic.adoc#_cypher[`@cypher` directive] and/or xref::/directives/custom-logic.adoc#_customresolver[Custom Resolvers].
+- Extensive xref::filtering.adoc[Filtering], xref::queries-aggregations/sorting.adoc[Sorting] and xref::/queries-aggregations/pagination.adoc[Pagination] options.
+- Options for xref::/directives/database-mapping.adoc[Database mapping] and value xref::/directives/autogeneration.adoc[Autogeneration].
+- xref::/security/index.adoc[Security options] and additional xref::schema-configuration/index.adoc[Schema Configuration].
+- A xref::getting-started/toolbox.adoc[Toolbox] (UI) to experiment with your Neo4j GraphQL API on Neo4j Desktop.
```

Comment:
I'm not sure if this is helpful here

This feels more like a "features" blurb to show off in the index, but we may want to refine a bit the list of features

There are some pretty big ones missing such as subscription and federation integration. And potentially Database mapping and autogeneration do not feel as big as to be part of the "main features" list?